doc/UsersGuide/source/_static/theme_overrides.css

```diff
@@ -0,0 +1,3 @@
+.wy-nav-content {
+   max-width: 100% !important;
+}
```

Comment:
Can we fix this?

Reply:
What is the concern with this? I set it so that the tables can be viewed without needing to scroll left-to-right as much. Without it, there's a stylish gray column on the right which looks nice but doesn't add functionality (see [here](https://sphinx-rtd-theme.readthedocs.io/en/stable/), as an example of the default).

Reply:
so this is the set regarding monitor fit: horizontal. No impact on pdf, right?

Reply:
Sorry, to be unclear. On the PR diff I saw a red-circle 'no newline' at line 3. That is all.

Reply:
It should only impact the horizontal fit on the screen. The !important just ensures that the horizontal fit of the wy-nav-content element overrides the theme default, which seems to be loaded after on RTD. That said, I have never succeeded in building the pdf version on my local system (for this or SRW), so I cannot run it to be 100% certain.

Reply:
@DeniseWorthen No worries! Just fixed it! :)